src/options.cpp

```diff
@@ -827,6 +827,10 @@ void initOptions()
                                    _("Adjust the volume of the music being played in the background."),
                                    0, 200, 100, COPT_CURSES_HIDE
                                   );
+    OPTIONS["SOUND_VOLUME"] = cOpt("graphics", _("Sound Volume"),
+                                   _("Adjust the volume of sound effects being played by the game."),
```

Comment:
That seems a bit confusing, what is the difference between "Sound Volume" and "Music Volume"? Isn't music some kind of sound? As you already describe it as "volume of sound effects", it should probably be named that way.

"Sound Volume" also lets me think it somehow affects the actual sounds in game.